compiler/README.md

```diff
@@ -0,0 +1,53 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This docuement discusses the ONE On-Device Neural Network compiler. The scope of this document is limited to Neural Network compiler which analyzes, quantizes and optimizes trained model. Model training, deployment & inference are NOT part of this. 
+
+---
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu 18.04.   
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Software/Package Requirments
+    * Cmake (ver??)
+    * g++ toolchain (ver??)
+
+<!-- Build Configuration -->
+* Configure  
+In-Progress
+
+<!-- Build -->
+* Build  
+In-Progress
+
+* Build Issues
+
+## Supported Neural Network Formats
+Neural Networks can be held in many formats & ONE supports muliple formats too.  
+But the preferred being the TensorFlow Lite format. ONE provides tools to convert other formats to TensorFlow Lite(??).   
+
+## Model Analysis
```

Comment:
Need help here.

Reply:
I'm not sure what you mean by analysis, maybe list some examples what you thought...

Reply:
@seanshpark Some kind of analysis useful for next step for optimization (like memory usage, an estimate of peak memory, etc to help in optimization/to be fed to optimizer so that optimizer can decide which code to put in cache, and how to allocate memory optimally)

Reply:
Currently you mentioned need development and not supported yet :)